ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -xeuo pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+  echo "Try to login..."
+  "${IBMCLOUD_CLI}" login -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function checkCli() {
+  export IBMCLOUD_CLI=ibmcloud
+  export IBMCLOUD_HOME=/output
+  echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
+  echo "Pause for debug...."
+  command -v ${IBMCLOUD_CLI}
+  ${IBMCLOUD_CLI} --version
+  ${IBMCLOUD_CLI} plugin list
+}
+
+checkCli
+region="${LEASED_RESOURCE}"
+export region
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export SSH_PUB_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-publickey
```

Comment:
Seem like `SSH_PUB_KEY_PATH` is not used in the script.

Reply:
I will remove it.